packages/amplify-app/src/index.js

```diff
@@ -329,10 +329,25 @@ async function createJSHelperFiles() {
   });
 }
 
+async function createAndroidHelperFiles() {
+  const configJsonData = '{"profile":"default", "envName":"amplify"}';
+  const configJsonObj = JSON.parse(configJsonData);
+  const configJsonStr = JSON.stringify(configJsonObj);
```

Comment:
why parsing first. Can't we use just an Object and stringify it

Reply:
I'm parsing so that the object is formatted properly before we stringify it. Otherwise there will be backslashes in the json file

Reply:
changed to json object